# Ilya J. Finkelstein (abridged CV)

<div class="research-image-container">
  <img src="/assets/images/members/Ilya-at-bench.jpg" align="right" class="research-image" width="400px" alt="IJF">
</div>
[MBB 3.422](https://maps.utexas.edu/buildings/utm/mbb)  
[Dept. of Molecular Biosciences](https://molecularbiosci.utexas.edu/)  
[University of Texas at Austin](https://www.utexas.edu/)  
Austin, TX  78712-1199  
lab: 	512-475-6172  
website: [https://finkelsteinlab.org](https://finkelsteinlab.org)    
e-mail: [ilya@finkelsteinlab.org](mailto: ilya@finkelsteinlab.org)  

## Education
**University of California, Berkeley (1997-2001)**  
B.S. Chemistry

**Stanford University (2001-2007)** <br>
Ph.D. Chemistry  
[(thesis: ~4 MB PDF)](/assets/pdfs/IJF_thesis_ultrafast_protein_dynamics.pdf)

## Professional Experience
  **Associate Professor (2019- )**
  [Department of Molecular Biosciences](https://molecularbiosci.utexas.edu/),  
  [Livestrong Cancer Institute](https://dellmed.utexas.edu/units/livestrong-cancer-institutes),  
  [Institute for Cellular and Molecular Biology](https://icmb.utexas.edu/),  
  [Center for Systems and Synthetic Biology](https://cssb.utexas.edu/),  
  [The University of Texas at Austin](https://www.utexas.edu/)
  **Assistant Professor (2012-2019)**  
  [Department of Molecular Biosciences](https://molecularbiosci.utexas.edu/),   
  [The University of Texas at Austin](https://www.utexas.edu/)

  **Postdoctoral Fellow (2007-2012)**  
  Columbia University Medical Center (with [Eric C. Greene](http://www.thegreenelab.com/))

  **Graduate Student (2001-2007)**  
  Stanford University (with [Michael D. Fayer](https://web.stanford.edu/group/fayer/))

## Awards and Honors
* Provost's Mentored Faculty Scholar
* Lorene Morrow Kelley Research Fellowship
* Norman Hackerman Award in Chemical Sciences, Welch Foundation (2021)
* Fellow, William H. Tonn Professorial Fund (2018)
* National Science Foundation CAREER Award (2015) 		
* American Federation of Aging Research Junior Fellow (2014)
* Emerging Investigator, Molecular BioSystems (2013)
* Protege, The Academy of Medicine, Engineering and Science of Texas (2013)
* CPRIT Fellow in Cancer Research (2012) 			
* NIH (K99/R00) Pathway to Independence Award (2011)
* NIH NRSA (F32) Postdoctoral Fellow (2007)			
* Veatch Graduate Fellowship, Stanford University (2005) 		
* McBain and Stauffer Graduate Fellowship, Stanford University (2003)  
* Chevron Research Fellowship (2000) 						

## Five Significant Publications (from [76+](/papers/))
* [**Massively Parallel Biophysical Analysis of CRISPR-Cas Complexes on Next Generation Sequencing Chips.**](/papers/paper/champ-cascade)  
Jung C.,* Hawkins JA.,* Jones SK.,* Xiao Y., Rybarski JR., Dillard KE., Hussmann J., Saifuddin FA., Savran CA., Ellington AD., Ke A., Press WH., Finkelstein IJ.   
(2017). _Cell_ **170**, 35–47.e13. (* equal contribution)

We developed a chip-hybridized affinity mapping platform (CHAMP) that recycles used NGS chips for massively-parallel biophysical studies of protein-nucleic acid interactions. Using CHAMP, we uncovered novel proofreading activities of a Type I-E CRISPR system on both synthetic and genomic DNA. We anticipate that this method will be broadly applicable for biophysical studies of protein-nucleic acid interactions.

* [**An aging-independent replicative lifespan in a symmetrically dividing eukaryote.**](/papers/paper/yeast-aging-multylm)  
Spivey EC.,* Jones SK.,* Rybarski JR., Saifuddin FA., Finkelstein IJ.   
(2017).  _eLife_ **6**, e20340. (* equal contribution)

This manuscript identifies fission yeast as the first eukaryotic cell that appears not to age. This remarkable observation was made possible by a new microfluidic device that we developed for monitoring thousands of individual fission yeast cells over their entire replicative lifespans. We show for the first time that these cells die via aging-independent mechanisms (e.g., genome instability). Remarkably, the replicative lifespan can be extended via drugs and genetic perturbations, suggesting that longevity can be extended without necessarily slowing aging. This work establishes fission yeast as a powerful model organism for cellular aging.


* [**Single-Molecule Imaging Reveals How Mre11-Rad50-Nbs1 Initiates DNA Break Repair.**](/papers/paper/mrn-dna-curtains)  
Myler LR.,* Gallardo IF.,* Soniat MM., Deshpande RA., Gonzalez XB., Kim Y., Paull TT., Finkelstein IJ. (2017).   
_Molecular Cell_ **67**, 891–898.e4.  

Mammalian DNA breaks are identified by the Mre11/Rad50/Nbs1 (MRN) and Ku70/Ku80 (Ku) complexes. Ku directs repair towards error-prone non-homologous end joining, whereas MRN promotes error-free homologous recombination. Here, we describe how MRN: (1) rapidly locates free DNA ends, (2) removes Ku from these ends, and (3) initiates homologous recombination. This work provides a molecular view light on how human cells initiate DNA break repair.

* [**Single-molecule imaging reveals the mechanism of Exo1 regulation by single-stranded DNA binding proteins.**](/papers/paper/exo1-ssb-curtains)  
Logan R. Myler*, Ignacio F. Gallardo*, Yi Zhou, Fade Gong, Soo-Hyun Yang, Marc S. Wold, Kyle M. Miller, Tanya T. Paull, Ilya J. Finkelstein. (2016).  
_Proc Natl Acad Sci USA._ **113**, E1170-9.

Exonuclease 1 (Exo1) is a conserved eukaryotic nuclease that participates in DNA break repair, telomere maintenance, and mismatch repair. Here we examined Exo1 activity on DNA and in the presence of single-stranded DNA binding proteins using high-throughput single-molecule fluorescence imaging. We report that both human and yeast Exo1 are processive nucleases but are rapidly turned over by replication protein A (RPA). In the presence of RPA, Exo1 retains limited DNA-processing activity, albeit via a distributive binding mechanism. RPA-depleted human cells show elevated Exo1 loading but reduced overall DNA resection, underscoring the many roles of RPA in regulating DNA resection in vivo.

* [**Dynamic DNA binding licenses a repair factor to bypass roadblocks in search of DNA lesions.**](/papers/paper/dna-bypass-roadblocks)  
Brown MW., Kim Y., Williams GM., Huck JD., Surtees JA., Finkelstein IJ. (2016).   
_Nature Communications_ **7**, 10607.

Post-replicative mismatch repair must identify rare DNA lesions (~1 mispair in 107-108 basepairs) in chromatin. Using single-molecule imaging, we demonstrate how two related eukaryotic mismatch sensors recognize these lesions on chromatin. Both factors can scan naked DNA, but only one of these complexes can also navigate on a chromatinized DNA substrate. This work begins to define the mechanisms of post-replicative mismatch repair on chromatin.

## Recent Professional Activities  
* ASCB Faculty Research and Eductaion Development Mentor (2021)
* Organizer: Molecular Biophysics Focus Group, Biophysical Society (2019)
* Austin Forum on Technology and Society co-panelist (2018)
* SXSW Interactive Conference co-panelist (2018)
* Member, Biophysical Society
* Journal Reviewer: Nature, Science, Cell, Molecular Cell, Cell Reports, eLife, Proceedings of the National Academies of Science, Nature Communications, Nature Chemical Biology, Nature Structural and Molecular Biology, Nucleic Acids Research, Journal of the American Chemical Society, Scientific Reports, Journal of Biological Chemistry, Biochemistry, J Phys Chem B, PloS ONE, Analytical Chemistry, Analytical Biochemistry, BioEssays, Methods
* Grant Reviewer: NSF, NIH, DHS, HHMI, French National Cancer Institute, Dutch Foundation for Fundamental Research


_Updated January 27, 2022_
