---
layout: paper
title: "Massively parallel kinetic profiling of natural and engineered CRISPR nucleases"
volume:
issue:
pages:
year: "2020"
shortref: "Jones_et al._ NatBiotech (2020)"
nickname: "NucleaSeq"
journal: "Nature Biotechnology"
authors: "Stephen K. Jones Jr.&#42;, John A. Hawkins&#42;, Nicole V. Johnson, Cheulhee Jung, Kuang Hu, James R. Rybarski, Janice S. Chen, Jennifer A. Doudna, William H. Press & Ilya J. Finkelstein; (&#42; co-first authors)"
pdf: 0063-2020-NatBiotech-Jones et al.pdf
supplement: 0063-2020-NatBiotech-Jones et al-supplement.pdf
doi: 10.1038/s41587-020-0646-5
pmcid:
pmid: 32895548
extra_text: ''
category: paper
tags:
---
{% include JB/setup %}

# Abstract
Engineered SpCas9s and AsCas12a cleave fewer off-target genomic sites than wild-type (wt) Cas9. However, understanding their fidelity, mechanisms and cleavage outcomes requires systematic profiling across mispaired target DNAs. Here we describe NucleaSeq-nuclease digestion and deep sequencing-a massively parallel platform that measures the cleavage kinetics and time-resolved cleavage products for over 10,000 targets containing mismatches, insertions and deletions relative to the guide RNA. Combining cleavage rates and binding specificities on the same target libraries, we benchmarked five SpCas9 variants and AsCas12a. A biophysical model built from these data sets revealed mechanistic insights into off-target cleavage. Engineered Cas9s, especially Cas9-HF1, dramatically increased cleavage specificity but not binding specificity compared to wtCas9. Surprisingly, AsCas12a cleavage specificity differed little from that of wtCas9. Initial DNA cleavage sites and end trimming varied by nuclease, guide RNA and the positions of mispaired nucleotides. More broadly, NucleaSeq enables rapid, quantitative and systematic comparisons of specificity and cleavage outcomes across engineered and natural nucleases.