---
layout: paper
title: "A mechanistic model improves off-target predictions and reveals the physical basis of SpCas9 fidelity"
volume:
issue:
pages:
year: "2020"
shortref: "Eslami-Mossallam_et al._BioRxiv (2020)"
nickname: "SpCas9-model"
journal: "BioRxiv"
authors: "Behrouz Eslami-Mossallam&#42;, Misha Klein&#42;, Constantijn v.d. Smagt, Koen v.d. Sanden, Stephen K. Jones Jr., John A. Hawkins, Ilya J. Finkelstein & Martin Depken (&#42; co-first authors)"
pdf: 2020-BioRxiv-Eslami-Mossallam et al.pdf
supplement: 2020-BioRxiv-Eslami-Mossallam et al-supplement.pdf
doi: 10.1101/2020.05.21.108613
pmcid:
pmid:
extra_text: ''
category: paper
tags:
---
{% include JB/setup %}

# Abstract
The SpCas9 endonuclease has become an important tool in gene-editing and basic science alike. Though easily programmed to target any sequence, SpCas9 also shows considerable activity over genomic off-targets. Many empirical facts regarding the targeting reaction have been established, but a comprehensive mechanistic description is still lackingâ€”limiting fundamental understanding, our ability to predict off-target activity, and ultimately the safe adaptation of the SpCas9 toolkit for therapeutics. By mechanistically modelling the SpCas9 structure-function relationship, we simultaneously capture binding and cleavage dynamics for SpCas9 and Sp-dCas9 in terms of free-energies. When our model is trained on high-throughput data, we outperform state-of-the-art off-target prediction tools. Based on the biophysical parameters we extract, our model predicts the open, intermediate, and closed complex configurations described in single-molecule FRET experiments, and indicates that R-loop progression is tightly coupled to structural changes in the targeting complex. We further show that SpCas9 targeting kinetics are tuned for extended sequence specificity while maintaining on-target efficiency. Our approach can be used to characterize any other CRISPR derived nuclease, and contrasting future studies of high-fidelity variants with the SpCas9 benchmark we here provide will help elucidate the determinants of CRISPR fidelity and the path to increased specificity and efficiency in engineered systems.