---
layout: paper
title: "A kinetic model improves off-target predictions and reveals the physical basis of SpCas9 fidelity"
volume:
issue:
pages:
year: "2020"
shortref: "Eslami-Mossallam_et al._BioRxiv (2020)"
nickname: "SpCas9-model"
journal: "BioRxiv"
authors: "Behrouz Eslami-Mossallam&#42;, Misha Klein&#42;, Constantijn v.d. Smagt, Koen v.d. Sanden, Stephen K. Jones Jr., John A. Hawkins, Ilya J. Finkelstein & Martin Depken (&#42; co-first authors)"
pdf: 2020-BioRxiv-Eslami-Mossallam et al.pdf
supplement: 2020-BioRxiv-Eslami-Mossallam et al-supplement.pdf
doi: 10.1101/2020.05.21.108613
pmcid:
pmid:
extra_text: ''
category: paper
tags:
---
{% include JB/setup %}

# Abstract
The S. pyogenes (Sp) Cas9 endonuclease is an important gene-editing tool. SpCas9 is directed to target sites via a single guide RNA (sgRNA). However, SpCas9 also binds and cleaves genomic off-target sites that are partially matched to the sgRNA. Here, we report a microscopic kinetic model that simultaneously captures binding and cleavage dynamics for SpCas9 and Sp-dCas9 in free-energy terms. This model not only outperforms state-of-the-art off-target prediction tools, but also details how Sp-Cas9’s structure-function relation manifests itself in binding and cleavage dynamics. Based on the biophysical parameters we extract, our model predicts SpCas9’s open, intermediate, and closed complex configurations and indicates that R-loop progression is tightly coupled with structural changes in the targeting complex. We show that SpCas9 targeting kinetics are tuned for extended sequence specificity while maintaining on-target efficiency. Our extensible approach can characterize any CRISPR-Cas nuclease – benchmarking natural and future high-fidelity variants against SpCas9; elucidating determinants of CRISPR fidelity; and revealing pathways to increased specificity and efficiency in engineered systems.