---
layout: paper
title: "Metagenomic Discovery of CRISPR-Associated Transposons"
volume: 118
issue: 49
pages:
year: "2021"
shortref: "Rybarski et al. PNAS (2021)"
nickname: "CAST metagenomic discovery"
journal: "PNAS"
authors: " James R. Rybarski&#42;, Kuang Hu&#42;, Alexis M. Hill&#42;, Claus O. Wilke, Ilya J. Finkelstein (&#42; co-first authors)"
pdf: 0074-2021-PNAS-Rybarski et al.pdf
supplement: 0074-2021-PNAS-Rybarski et al-supplement.pdf
doi: 10.1073/pnas.2112279118
pmcid:
pmid: 
extra_text: ''
category: paper
tags: [gene editing, bioinformatics, CAST, transposition, CRISPR RNA]
---
{% include JB/setup %}

# Abstract
'CRISPR-associated Tn7 transposons (CASTs) co-opt cas genes for RNA-guided transposition. CASTs are exceedingly rare in genomic databases; recent surveys have reported Tn7-like transposons that co-opt Type I-F, I-B, and V-K CRISPR effectors. Here, we expand the diversity of reported CAST systems via a bioinformatic search of metagenomic databases. We discover architectures for all known CASTs, including arrangements of the Cascade effectors, target homing modalities, and minimal V-K systems. We also describe families of CASTs that have co-opted the Type I-C and Type IV CRISPR-Cas systems. Our search for non-Tn7 CASTs identifies putative candidates that include a nuclease dead Cas12. These systems shed light on how CRISPR systems have coevolved with transposases and expand the programmable gene-editing toolkit.'

