---
layout: paper
title: "Antibody escape and cryptic cross-domain stabilization in the SARS-CoV-2 Omicron spike protein"
volume: 30
issue:
pages: 1-30
year: "2022"
shortref: "Javanmardi et al. CellHostMicrobe (2022)"
nickname: "Ab-escape-Omicron-spike"
journal: "CellHostMicrobe"
authors: "Kamyab Javanmardi&dagger;, Thomas H. Segall-Shapiro, Chia-Wei Chou, Daniel R. Boutz, Randall J. Olsen, Xuping Xie, Hongjie Xia, Pei-Yong Shi, Charlie D. Johnson, Ankur Annapareddy, Scott Weaver, James M. Musser, Andrew D. Ellington, Ilya J. Finkelstein&dagger;, Jimmy D. Gollihar&dagger;(&dagger; co-corresponding)"
pdf: 0080-2022-CellHostMicrobe-Javanmardi et al.pdf
supplement: 0080-2022-CellHostMicrobe-Javanmardi et al-supplement.pdf
doi: 10.1016/j.chom.2022.07.016
pmcid: PMC9350683
pmid: 35988543
extra_text: ''
category: paper
tags:
---
{% include JB/setup %}

# Abstract
The worldwide spread of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has led to the repeated emergence of variants of concern. For the Omicron variant, sub-lineages BA.1 and BA.2, respectively, contain 33 and 29 nonsynonymous and indel spike protein mutations. These amino acid substitutions and indels are implicated in increased transmissibility and enhanced immune evasion. By reverting individual spike mutations of BA.1 or BA.2, we characterize the molecular effects of the Omicron spike mutations on expression, ACE2 receptor affinity, and neutralizing antibody recognition. We identified key mutations enabling escape from neutralizing antibodies at a variety of epitopes. Stabilizing mutations in the N-terminal and S2 domains of the spike protein can compensate for destabilizing mutations in the receptor binding domain, enabling the record number of mutations in Omicron. Our results provide a comprehensive account of the mutational effects in the Omicron spike protein and illustrate previously uncharacterized mechanisms of host evasion.