---
layout: paper
title: "Unraveling the mechanisms of PAMless DNA interrogation by SpRY Cas9"
volume: 
issue:
pages:
year: "2023"
shortref: "Hibshman et al. BioRxiv (2023)"
nickname: "PAMless-DNA-SpRY-Cas9"
journal: "BioRxiv"
authors: "Grace N. Hibshman&#42;, Jack P. K. Bravo&#42;&dagger;, Hongshan Zhang, Tyler L. Dangerfield, Ilya J. Finkelstein, Kenneth A. Johnson, David W. Taylor&dagger; (&#42; co-first authors) (&dagger; co-corresponding)"
pdf: 2023-BioRxiv-Hibshman et al.pdf
supplement: 
doi: 10.1101/2023.06.22.546082
pmcid:
pmid: 
extra_text: ''
category: paper
tags: 
---
{% include JB/setup %}

# Abstract
CRISPR-Cas9 is a powerful tool for genome editing, but the strict requirement for an “NGG” protospacer-adjacent motif (PAM) sequence immediately adjacent to the DNA target limits the number of editable genes. To overcome the PAM requirement, a recently developed Cas9 variant, called SpRY-Cas9 was engineered to be “PAMless” (1, 2). However, the molecular mechanisms of how SpRY can recognize all potential PAM sequences and still accurately identify DNA targets have not been investigated. Here, we combined enzyme kinetics, cryo-EM, and single-molecule imaging to determine how SpRY interrogates DNA and recognizes target sites for cleavage. Divergent PAM sequences can be accommodated through conformational flexibility within the PAM-interacting region of SpRY, which facilitates tight binding to off-target DNA sequences. Once SpRY correctly identifies a target site, nuclease activation occurs ∼1,000-fold slower than for Streptococcus pyogenes Cas9, enabling us to directly visualize multiple on-pathway intermediate states. Insights gained from our intermediate structures prompted rationally designed mutants with improved DNA cleavage efficiency. Our findings shed light on the molecular mechanisms of PAMless genome editing with SpRY and provide a framework for the design of future genome editing tools with improved versatility, precision, and efficiency.