---
layout: paper
title: "Distinct horizontal transfer mechanisms for type I and type V CRISPR-associated transposons"
volume: 
issue:
pages:
year: "2023"
shortref: "Hu et al. BioRxiv (2023)"
nickname: "horizontal-transfer"
journal: "BioRxiv"
authors: "Kuang Hu&dagger;, Chia-Wei Chou, Claus O. Wilke, Ilya J. Finkelstein&dagger; (&dagger; co-corresponding) "
pdf: 2023-BioRxiv-Hu et al.pdf
supplement: 2023-BioRxiv-Hu et al-supplement.pdf
doi: 10.1101/2023.03.03.531003
pmcid:
pmid: 
extra_text: ''
category: paper
tags: gene editing, transposition, bacterial defense, selfish genetic element
---
{% include JB/setup %}

# Abstract
CRISPR-associated transposons (CASTs) have co-opted CRISPR-Cas proteins and Tn7-family transposons for RNA-guided vertical and horizontal transmission. CASTs encode minimal CRISPR arrays but lack all spacer acquisition genes. Here, we define how different CASTs target new invading mobile elements without updating their own CRISPR arrays. A bioinformatic analysis reveals that all CAST sub-families co-exist with defense-associated CRISPR-Cas systems. Using a quantitative transposition assay, we show that type I-F and I-B CASTs use CRISPR RNAs (crRNAs) from these defense systems for horizontal gene transfer. A high-resolution structure of the type I-F CAST-Cascade in complex with a type III-B crRNA reveals a sequence-independent mechanism for direct repeat recognition. Type I CASTs recognize heterologous CRISPR arrays via a short hairpin in the direct repeat of their crRNA. In contrast, type V CASTs require the Cas12k effector protein but not any crRNA for unguided transposition. This transposition causes random genomic insertions via a copy-and-paste mechanism, even with over-expression of the S15 co-factor. Conversely, a single guide RNA, in concert with S15, increases on-target integration for type V CASTs. These discoveries explain how CASTs horizontally transfer to new hosts without updating their own CRISPR arrays. More broadly, this work will guide further efforts to engineer the activity and specificity of CASTs for gene editing applications.